de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/Title.java

```diff
@@ -100,7 +100,7 @@ public Title(String titleText) throws WikiTitleParsingException  {
             this.disambiguationText = null;
         }
 
-        if (getEntity() == null) {
+        if (getEntity() == null || (getEntity() != null && getEntity().isEmpty())) {
```

Comment:
Maybe could use Apache Commons `CollectionUtils.isEmpty(Collection<?>)` instead of calling `getEntity()` three times in a row?

Reply:
@reckart That's a good hint. I will it this in a couple of minutes.